src/components/content-tab-assistant.tsx

```diff
@@ -198,15 +199,23 @@ export function ContentTabAssistant( { selectedSite }: ContentTabAssistantProps
 					<UnauthenticatedView onAuthenticate={ authenticate } />
 				) }
 			</div>
-			<AIInput
-				disabled={ disabled }
-				input={ input }
-				setInput={ setInput }
-				handleSend={ () => handleSend() }
-				handleKeyDown={ handleKeyDown }
-				clearInput={ clearInput }
-				isAssistantThinking={ isAssistantThinking }
-			/>
+
+			<div className="bg-white flex flex-col">
+				<AIInput
+					disabled={ disabled }
+					input={ input }
+					setInput={ setInput }
+					handleSend={ () => handleSend() }
+					handleKeyDown={ handleKeyDown }
+					clearInput={ clearInput }
+					isAssistantThinking={ isAssistantThinking }
+				/>
+				<div data-testid="guidelines-link" className="text-gray-500 self-end p-2">
+					{ createInterpolateElement( __( 'Powered by exerimental AI. <a>Learn More.</a>' ), {
```

Comment:
There’s a typo in experimental.

```suggestion
					{ createInterpolateElement( __( 'Powered by experimental AI. <a>Learn More.</a>' ), {
```

Reply:
Changed as requested! Thanks for finding this!